cron/calculateAssessmentQuestionStats.sql

```diff
@@ -0,0 +1,158 @@
+-- BLOCK all
+WITH mean_question_scores AS (
```

Comment:
I'd like to make this a `FUNCTION` that takes an `assessment_id` as an argument and only updates the statistics for that assessment. Then we can call it for each assessment in the cron job.